### PR TITLE
Translate views to hybrid Polymer 1 and 2

### PR DESCRIPTION
**Pull request by warpech**
@alshakero pls review

This PR also includes a fix for the incorrect HTML markup, which was detected by the ViewTranslator

### CODE REVIEW

File: src/SampleWebsiteTheme/wwwroot/SampleWebsiteTheme/html/DesktopSurface.html [hunk 1]
```diff
@@ -40,14 +40,14 @@
...
         <starcounter-include slot="samplewebsitetheme/topbar-leftcorner" view-model="{{model.Sections.TopBarLeftCorner}}" is-visible$="[[_showMainMenu]]" on-click="toggleMenu"></starcounter-include>
         <starcounter-include slot="samplewebsitetheme/topbar-left" class="desktop-theme__top-left" view-model="{{model.Sections.TopBarLeft}}"></starcounter-include>
         <starcounter-include slot="samplewebsitetheme/topbar-center" view-model="{{model.Sections.TopBarCenter}}" class="desktop-theme__top-center"></starcounter-include>
         <starcounter-include slot="samplewebsitetheme/topbar-right" class="desktop-theme__top-right" view-model="{{model.Sections.TopBarRight}}"></starcounter-include>
         <starcounter-include slot="samplewebsitetheme/main" view-model="{{model.Sections.Main}}"></starcounter-include>
-    </template>
+    </template></dom-bind>
     <script>
         (function () {
             var template = (document._currentScript || document.currentScript).previousElementSibling;
```
Comment (alshakero) [inline src/SampleWebsiteTheme/wwwroot/SampleWebsiteTheme/html/DesktopSurface.html]: Shouldn't this be changed to 
```js
var script = document._currentScript || document.currentScript;
var template = Polymer.Element ? script.previousElementSibling : script.previousElementSibling.firstElementChild;
```

File: src/SampleWebsiteTheme/wwwroot/SampleWebsiteTheme/html/Slogan.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
-﻿<template>
-    <template is="dom-bind">
+<template>
+    <dom-bind><template is="dom-bind">
         Our app is the best
```
Comment (alshakero) [inline src/SampleWebsiteTheme/wwwroot/SampleWebsiteTheme/html/Slogan.html]: Might be out of this PR's scope, but why have a `dom-bind` at all when the content is static?

File: src/SampleWebsiteTheme/wwwroot/SampleWebsiteTheme/html/TermsAndConditionsLink.html [hunk 3]
```diff
@@ -1,5 +1,5 @@
-﻿<template>
-    <template is="dom-bind">
+<template>
+    <dom-bind><template is="dom-bind">
         <a href="/TermsAndConditions">Terms and Conditions</a>
```
Comment (alshakero) [inline src/SampleWebsiteTheme/wwwroot/SampleWebsiteTheme/html/TermsAndConditionsLink.html]: Most probably I'm missing the point, but it's worth commenting: same as above.

**Comment by warpech** [comment]
Thanks for the review!

**Comment by warpech** [comment]
Setting as “Finished” because it is:
- released (with release notes) https://github.com/Starcounter/SampleWebsiteTheme/releases/tag/2.1.0
- published in the App Warehouse

**Pull request merged**